src/reducers/stage-size.js

```diff
@@ -32,6 +28,5 @@ const setStageSize = function (stageSize) {
 export {
     reducer as default,
     initialState as stageSizeInitialState,
-    setStageSize,
-    STAGE_SIZES
```

Comment:
I suggest exporting another enum called `STAGE_SIZE_MODES` which is only large and small, to distinguish from the `STAGE_SIZES` (or `STAGE_DISPLAY_SIZES`?).